libs/portals/admin/signature-collection/src/screens/List/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { IntroHeader, PortalNavigation } from '@island.is/portals/core'
+import { useLoaderData, useRevalidator } from 'react-router-dom'
+import { useEffect } from 'react'
+import { SignatureCollectionList } from '@island.is/api/schema'
+import { signatureCollectionNavigation } from '../../lib/navigation'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../lib/messages'
+import {
+  Box,
+  Button,
+  GridColumn,
+  GridContainer,
+  GridRow,
+  Hidden,
+  Input,
+  Stack,
+} from '@island.is/island-ui/core'
+import Signees from './components/signees'
+import PaperUpload from './components/paperUpload'
+import header from '../../../assets/headerImage.svg'
+import format from 'date-fns/format'
+
+const List = () => {
+  const { list } = useLoaderData() as { list: SignatureCollectionList }
+  const { revalidate } = useRevalidator()
+  const { formatMessage } = useLocale()
+
+  useEffect(() => {
+    revalidate()
+  }, [])
+
+  return (
+    <GridContainer>
+      <GridRow direction="row">
+        <GridColumn
+          span={['12/12', '5/12', '5/12', '3/12']}
+          offset={['0', '7/12', '7/12', '0']}
+        >
+          <PortalNavigation
+            navigation={signatureCollectionNavigation}
+            title={formatMessage(m.signatureListsTitle)}
+          />
+        </GridColumn>
+        <GridColumn
+          paddingTop={[5, 5, 5, 2]}
+          offset={['0', '0', '0', '1/12']}
+          span={['12/12', '12/12', '12/12', '8/12']}
+        >
```

Comment:
Looks like same layout as in `AllLists` and `List` so you could use[ similar pattern](https://github.com/island-is/island.is/blob/main/libs/portals/admin/ids-admin/src/module.tsx#L75) to what we did in IDS Admin for tenant. We created a empty parent route which is the shared layout and uses the `<Outlet>` component from react router. Then nested routes are rendered in the outlet.